files/.condarc

```diff
@@ -1,8 +1,12 @@
 channels:
   - conda-forge
   - r
-  - bioconda
+  - moj-analytical-services
   - defaults
 
 envs_dirs:
   - ~/.conda/envs
+
+pinned_packages:
+  - openssl=1.1
+  - r-base=3.5.1
```

Comment:
Will this blow up if they remove this and add a newer version?

Reset the conda environment with the new command should still work anyway.

I'm just curious of why you're pinning these. Sounds sensible as if my understanding of this is correct we'll know that a user using the chart version foo will run R version bar.

Reply:
I'm not sure it will blow up actually, or even if it really works - I'm still testing it out, I think the base environment is pinned but I'm not sure about the rstudio env they create :confused: 

The intention is that I don't want them inadvertently downgrading their R version if they install an old library that depends on R3.4 
The reason for pinning openssl is that if it's changed to a version that Rstudio wasn't compiled with then breaks things like `install.packages`